src/Components/Aspire.Microsoft.Orleans.Client/Aspire.Microsoft.Orleans.Client.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you ensure everything in the [checklist](https://github.com/dotnet/aspire/blob/main/src/Components/README.md#checklist-for-new-components) is done for these new components?